include/cpp2util.h

```diff
@@ -882,6 +882,12 @@ inline constexpr auto is( auto const& x, auto const& value ) -> bool
     return false;
 }
 
+// Free function predicate case
+template< typename X >
+inline constexpr auto is( X const& x, bool (&value)(X const&) ) -> bool
```

Comment:
```suggestion
constexpr auto is( X const& x, bool (&value)(X const&) ) -> bool
```
A template function is implicitly `inline`.

Reply:
Oh, yes. I forgot to remove it. Thank you!